src/tactic/ui/panel/table_layout_wdg.py

```diff
@@ -7095,6 +7095,25 @@ def handle_load_behaviors(self, table):
 }
 
 
```

Comment:
Things to consider:

Are you looking at insert row (-1 ) row at any point 

What if there is no parent group?